Template/Trigger/UserInteractionTrigger.web.js

```diff
@@ -0,0 +1,30 @@
+(function () {
+    return function (parameters, TagManager) {
+        this.setUp = function (triggerEvent) {
+
+            const eventNames = [
```

Comment:
@JaJuMa Nothing wrong here, but as far as I remember this will create few problem in minification as per our code
Can you change it to `var`?